lib/graphql/schema.ts

```diff
@@ -295,8 +295,8 @@ export type CardLimit = {
 };
 
 export type CardLimitInput = {
-  maxAmountCents: Scalars['Float'];
-  maxTransactions: Scalars['Float'];
+  maxAmountCents?: InputMaybe<Scalars['Float']>;
+  maxTransactions?: InputMaybe<Scalars['Float']>;
```

Comment:
Maybe it shouldn't be float.
Not a blocker if it works.

Reply:
🤔 it's automatically generated type and on BE it's number